src/plugins/intel_gpu/src/graph/include/generic_layer.hpp

```diff
@@ -37,11 +37,15 @@ struct generic_layer : public primitive_base<generic_layer> {
     const kernel_selector::generic_kernel_params generic_params;
 
     size_t hash() const override {
-        size_t seed = primitive::hash();
+        size_t seed = 0;
```

Comment:
Why should we change the basic seed to 0?

Reply:
Modify hash and ==operator to use genric_params instead of id and remove zero seed